### ISSUE TITLE
Easy re-branding of the app

### ISSUE THREAD

**Issue by ajburls** [dwyl/best-evidence#107]
As a product owner
I want to be able to "re-brand" the app simply and quickly for different sponsors of the use of the app

This point follows on from issue #36 where we are currently talking about branding the app for City University.

Could changing a logo and colours on front page and about us page  be set up in such a way that this could easily be done?

**Comment by markwilliamfirth** [comment]
@ajburls how many different rebrands would you estimate would need to occur? i.e. how many sponsors / different versions would there be?

**Comment by ajburls** [comment]
@markwilliamfirth  Hard to say.  I can imagine doing lots of separate projects with different specialities or groups e.g. RCN, College of Optometrist, dentists, radiographers, students of mental health nursing, a particular hospital or practice...

Also homologous organisations from other countries

In theory many hundreds (each branding linked to/by the flag/s for specialty or organisation we discussed elsewhere)

In practice, I wouldn't expect to use more than ten, probably a lot less, in the next couple of years as each would be linked to a separate project.

**Comment by markwilliamfirth** [comment]
@ajburls ok if it's less than 10 and it's not an immediate requirement then it's probably best to leave this issue in the backlog and us to manually create the new designs as they come in. We could create some kind of content management system (CMS) to do the above, but I don't think the time investment would pay off for the level of customisation required.

**Comment by ajburls** [comment]
@markwilliamfirth  The problem with that solution is that if you have to do it we need to find money each time and that would be a barrier. Can you give me a ballpark figure for what each customisation would be likely to cost (I won't hold you to it ;-))

**Comment by markwilliamfirth** [comment]
@ajburls yes we can do that - we should probably discuss this one at sprint 3 sprint planning as the initial rebranding has already been completed and this isn't currently a priority
